dtfs-central-api/src/v1/controllers/bank/get-banks.controller.ts

```diff
@@ -0,0 +1,51 @@
+import { Response } from 'express';
+import { HttpStatusCode } from 'axios';
+import { CustomExpressRequest, Bank, ApiError, ApiErrorResponseBody } from '@ukef/dtfs2-common';
+import { getAllBanks } from '../../../repositories/banks-repo';
```

Comment:
I'm finding having this new TS controller and the existing JS controller with the same name confusing, which controller is used for what?

Reply:
They don't have the same name as far as I can see - one is called `get-banks.controller` and the other is called `get-bank.controller`